draft-bonaventure-quic-atsss-overview.mkd

```diff
@@ -709,19 +704,19 @@ Both proposals rely upon single QUIC connections and inherit thus the same issue
 
 ## Encapsulation Overhead {#sec-overhead}
 
-In 3GPP architectures, a variety of encapsulations are already used to carry user data. The use of QUIC as a tunneling method for ATSSS purposes will add an extra overhead. This overhead is even exacerbated when the user data is forwarded over a non-3GPP network access given that an IPsec tunnel is used to graft the UE to the 5G Core Network. For example, the overhead without taking into account the QUIC tunnel is at least 142 octets for an IPv6 packet forwarded over a non-3GPP network access. This compares unfavorably to ATSSS Phase 1, which does not add any overhead. 
+In 3GPP architectures, a variety of encapsulations are already used to carry user data. The use of QUIC as a tunneling method for ATSSS will add some additional overhead. When the user data is forwarded over a non-3GPP network access, this overhead comes further in addition to an IPsec tunnel between the UE to the 5G Core Network which is already at least 142 octets for an IPv6 packet forwarded over a non-3GPP network access. In contrast, the MPTCP based mechanism in ATSSS Phase 1, does only add some minor overheading during connection establishment, but addiotional over during the rest of the connection. 
 
 More investigation is required to assess whether the ATSSS Phase 2 overhead is an issue. Solutions to optimize that overhead could be considered, if needed. 
 
 ## Multiple Encryptions
 
-QUIC data that matches an ATSSS rule will be subject to multiple instances of encryptions. Some of these encryptions are redundant (e.g., ATSSS encryption vs. IP Encapsulating Security Payload (ESP)).
+The use of QUIC as a tunneling protocol in ATSSS with add an additional layer of encryption. As there is already encryptions on other layers (e.g.  IP Encapsulating Security Payload (ESP)) this leads to multiple layer of encryption with might be redundant.
```

Comment:
The use of QUIC as a tunneling protocol in ATSSS will add an additional layer of encryption. As there is already encryptions on other layers (e.g., IP Encapsulating Security Payload (ESP)) this leads to multiple layers of encryption with might be redundant.

Reply:
typo